release-notes/5.0/5.0.6/5.0.6.md

```diff
@@ -0,0 +1,400 @@
+# .NET 5.0.6 - May 11, 2021
+
+The .NET 5.0.6 and .NET SDK 5.0.202 releases are available for download. The latest 5.0 release is always listed at [.NET 5.0 Releases](../README.md).
```

Comment:
There's a few others that aren't updated to 5.0.203 in this file too, but not sure if that's intentional.

```suggestion
The .NET 5.0.6 and .NET SDK 5.0.203 releases are available for download. The latest 5.0 release is always listed at [.NET 5.0 Releases](../README.md).
```